tools/ZAPD/ZAPD/genbuildinfo.py

```diff
@@ -2,7 +2,6 @@
 
 import argparse
 from datetime import datetime
-import getpass
 import subprocess
```

Comment:
similarly, this tool is located at https://github.com/NEstelami/ZAPD/ and should have changes made upstream instead of here